podaac/subsetter/subset.py

```diff
@@ -1077,9 +1077,39 @@ def walk_h5py(data_new, group):
     return nc_dataset, has_groups
 
 
+def get_coord_vars(dataset, lat_var_names=None, lon_var_names=None, time_var_names=None):
```

Comment:
Just a minor code comprehension suggestion. We now have `get_coord_vars` and `get_coord_variable_names` functions which seem at first look to do the same thing. Is there a better name for these functions that more accurately reflects their purpose? Maybe `get_coordinate_variable_names` for this function and `find_coordinate_variable_names` or `compute_coordinate_variable_names` for the other?

Reply:
Thanks @frankinspace , I agree this makes a lot more sense. I've renamed them as follows:

* `get_coord_vars` -> `get_coord_variable_names`
* `get_coord_variable_names` -> `compute_coordinate_variable_names`
* `get_time_variable_name` -> `compute_time_variable_name`